src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -142,14 +143,25 @@ def wrapped(func):
 
 class CliCommand(object):
 
-    def __init__(self, name, handler, description=None, table_transformer=None):
+    def __init__(self, name, handler, description=None, table_transformer=None,
+                 arguments_loader=None, description_loader=None):
```

Comment:
```
cli_command_with_handler(__name__, 'vm delete', of_vm_delete, cf_vm)
```
It's only the addition of the __name__ param that is different.

I'll be moving all commands to this form.
We don't want other command modules to do things incorrectly as the whole CLI will be slowed down.